src/MonitoringDemo/DirectoryEx.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace MonitoringDemo
+{
+    using System;
+    using System.IO;
+    using System.Threading;
+
+    static class DirectoryEx
+    {
+        public static void Delete(string directoryPath)
+        {
+            for (int i = 0; i < 3; i++)
+            {
+                try
+                {
+                    Directory.Delete(directoryPath, true);
+                    return;
+                }
+                catch (DirectoryNotFoundException)
+                {
+                    return;
+                }
+                catch (Exception)
+                {
+                    // ignored
+                    Thread.Sleep(5000);

```

Comment:
Why?

Reply:
When I wrote the powershell script I detected that especially the `.db` folder remains locked for a period of time by RavenDB. This makes sure things can get properly deleted in the folder. What I saw though is that the empty folder `.db` stays around nonetheless.

Reply:
@SzymonPobiega I find a better solution. Please review the latest changes and smoke test

Reply:
Some tested. Works like a charm. You rock man 👍 💯